modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -88,7 +88,7 @@ This section provides the necessary details that enable you to control egress tr
 |443
 |Provides core container images as a fallback when quay.io is not available.
 
-|`.q1w2.quay.rhcloud.com`
+|`*.q1w2.quay.rhcloud.com`
```

Comment:
In the AWS network firewall, the wildcard domain format is like '.q1w2.quay.rhcloud.com'; In the AWS DNS firewall, the wildcard domain format is like '*.q1w2.quay.rhcloud.com'. 

And synced with Will, the Network Firewall here is pointed to the AWS network firewall.

Reply:
@yasun1 Thanks for your input. So, I am revoking the change here.

Reply:
This section is for Firewall preq. So if we provide the list for AWS Firewall rules, we should use "." for all wild card hostnames instead of "*." as a consistent format.